02-overview-surveys.Rmd

```diff
@@ -108,63 +108,35 @@ When developing the questionnaire, it can be helpful to first outline the topics
 
-Additionally, when designing questions, researchers should consider the mode for the survey and adjust language appropriately. In self-administered surveys (e.g., web or mail), respondents can see all the questions and response options, but that is not the case in interviewer-administered surveys (e.g., CATI or CAPI). With interviewer-administered surveys, the response options need to be read aloud to the respondents, so the question may need to be adjusted to allow a better flow to the interview. Additionally, with self-administered surveys, because the respondents are viewing the questionnaire, the formatting of the questions is even more important to ensure accurate measurement. Incorrect formatting or wording can result in *measurement error*, so following best practices or using existing validated questions can reduce error. There are multiple resources to help researchers draft questions for different modes [e.g., @dillman2014mode; @Fowler1989; @Bradburn2004; @Tourangeau2004spacing].
+Additionally, when designing questions, researchers should consider the mode for the survey and adjust language appropriately. In self-administered surveys (e.g., web or mail), respondents can see all the questions and response options, but that is not the case in interviewer-administered surveys (e.g., CATI or CAPI). With interviewer-administered surveys, the response options need to be read aloud to the respondents, so the question may need to be adjusted to create a better flow to the interview. Additionally, with self-administered surveys, because the respondents are viewing the questionnaire, the formatting of the questions is even more important to ensure accurate measurement. Incorrect formatting or wording can result in *measurement error*, so following best practices or using existing validated questions can reduce error. There are multiple resources to help researchers draft questions for different modes [e.g., @dillman2014mode; @Fowler1989; @Bradburn2004; @Tourangeau2004spacing].
 
 #### Example: Number of Pets in a Household {.unnumbered #overview-design-questionnaire-ex}
 
 As part of our survey on the average number of pets in a household, researchers may want to know what animal most people prefer to have as a pet. Let's say we have the following question in our survey:
 
-<!-- Put this in as an image instead of as a block quote -->
-
-> What animal do you prefer to have as a pet?
->
-> <ul class="ro">
->
-> <li class="ro">
->
-> Dogs
->
-> </li>
->
-> <li class="ro">
->
-> Cats
->
-> </li>
->
-> </ul>
+```{r}
```

Comment:
These are huge images, will need to adjust the size on these (especially for PDF as it is off the page), but they are in!

Reply:
Resized :)